### ISSUE TITLE
filament-daterangepicker.css breaks custom theme

### ISSUE THREAD

**Issue by maxacarvalho** [malzariey/filament-daterangepicker-filter#140]
Hi,

After adding the `@import '/vendor/malzariey/filament-daterangepicker-filter/resources/css/filament-daterangepicker.css';` line to my custom theme the frontend breaks.  
I can confirm that after removing it everything works as expected.

The custom theme file is very simple:
```css
@import '/vendor/filament/filament/resources/css/theme.css';
@import '/vendor/malzariey/filament-daterangepicker-filter/resources/css/filament-daterangepicker.css';

@config 'tailwind.config.js';
```

**Comment by malzariey** [comment]
Can you share a reproduction repository

**Comment by malzariey** [comment]
> Hey @malzariey sorry about the delay. There you go https://github.com/maxacarvalho/l11-filament-fresh

Sorry for the delay. And Thank you for raising the issue.

Issue has been solved, in version 3.1.1
